data/init.sql

```diff
@@ -0,0 +1,334 @@
+(100, 'Candy Sale $.75 OFF', 'Limit one unit per coupon', '0.75', 'Dollar'),
+(101, 'NABISCO Cooke Sale $1 OFF', 'Limit one unit per coupon', '1', 'Dollar'),
+(102, 'Peter Pan Peanut Butter Half Off', '50% OFF on all sizes', '0.5', 'Percent'),
+(103, 'PRINGLES $.50 OFF', '', '0.5', 'Dollar'),
+(104, 'Planters 30% OFF single item', 'only items listed are eligible ', '0.3', 'Percent'),
+(105, 'Cookie Sale 25% OFF', 'All Cookies', '0.25', 'Percent');
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `PromotionItem`
+--
+
+DROP TABLE IF EXISTS `PromotionItem`;
+CREATE TABLE IF NOT EXISTS `PromotionItem` (
+  `ID` int(5) NOT NULL AUTO_INCREMENT,
+  `PromoCode` int(5) NOT NULL,
+  `ItemNumber` varchar(7) COLLATE latin1_general_ci NOT NULL,
+  `SalePrice` varchar(10) COLLATE latin1_general_ci NOT NULL,
+  PRIMARY KEY (`ID`)
+) ENGINE=MyISAM  DEFAULT CHARSET=latin1 COLLATE=latin1_general_ci AUTO_INCREMENT=54 ;
+
+--
+-- Dumping data for table `PromotionItem`
+--
+
+INSERT INTO `PromotionItem` (`ID`, `PromoCode`, `ItemNumber`, `SalePrice`) VALUES
+(1, 100, '1200004', '3.17'),
+(2, 100, '1203936', '2.10'),
+(3, 100, '1204035', '0.55'),
+(4, 100, '1204060', '1.51'),
+(5, 100, '1204100', '9.10'),
+(6, 100, '1204111', '1.57'),
+(7, 100, '1200420', '3.99'),
+(8, 100, '1200424', '1.23'),
+(9, 100, '1200553', '1.32'),
+(10, 100, '1204242', '1.55'),
+(11, 100, '1200653', '0.67'),
+(12, 100, '1204383', '3.11'),
+(13, 100, '1204410', '2.43'),
+(14, 100, '1200655', '0.44'),
+(15, 100, '1204430', '1.35'),
+(16, 100, '1200659', '0.93'),
+(17, 100, '1204436', '8.45'),
+(18, 100, '1206454', '3.40'),
+(19, 100, '1200025', '1.80'),
+(20, 100, '1203091', '2.63'),
+(21, 101, '1200200', '3.45'),
+(22, 101, '1200201', '3.47'),
+(23, 101, '1200383', '3.20'),
+(24, 101, '1219119', '3.71'),
+(25, 102, '1218660', '1.57'),
+(26, 103, '1204585', '3.85'),
+(27, 103, '1219418', '1.16'),
+(28, 103, '1219427', '1.24'),
+(29, 103, '1219884', '2.20'),
+(30, 103, '1200572', '1.56'),
+(31, 104, '1206078', '1.13'),
+(32, 104, '1203517', '3.33'),
+(33, 104, '1203530', '2.21'),
+(34, 105, '1200143', '2.21'),
+(35, 105, '1200200', '3.33'),
+(36, 105, '1200201', '3.35'),
+(37, 105, '1200383', '3.15'),
+(38, 105, '1204282', '7.61'),
+(39, 105, '1219119', '3.53'),
+(40, 105, '1200623', '1.61'),
+(41, 105, '1200662', '2.49'),
+(42, 105, '1200808', '5.06'),
+(43, 105, '1200825', '2.91'),
+(44, 105, '1200256', '2.39'),
+(45, 105, '1203212', '2.15'),
+(46, 105, '1206558', '1.82'),
+(47, 105, '1208093', '4.34'),
+(48, 105, '1201788', '3.84'),
+(49, 105, '1220270', '2.23'),
+(50, 105, '1220290', '2.64'),
+(51, 105, '1203567', '3.49'),
+(52, 105, '1202691', '3.47'),
+(53, 105, '1220616', '2.00');
+
+/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
+/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
+/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
```

Comment:
This is the file given to us on blackboard to create the database and tables associated with it. We can place it right in here and theoretically it is supposed to run and create everything for us if the database connection is right.